embassy-rp/Cargo.toml

```diff
@@ -25,6 +25,7 @@ critical-section-impl = ["critical-section/restore-state-u8"]
 unstable-pac = []
 
 time-driver = []
+pwm-driver = []
```

Comment:
No need to feature gate it.

(`time-driver` is optional because there can only be one `embassy-time` driver in a given binary, build fails otherwise. The feature is needed if the user wants to use `embassy-rp` but use something else as a time driver. This is not the case for PWM)